webpack/protocol-compat.js

```diff
@@ -0,0 +1,2 @@
+// Necessary until we upgrade protocol to include this fix: https://github.com/mozilla/protocol/issues/687
+window.Mzp = {};
```

Comment:
Could you file an issue to update Protocol that will also reference this file/issue?

Reply:
https://github.com/mozilla/sumo-project/issues/892